docs/operating-eck/uninstalling-eck.asciidoc

```diff
@@ -7,18 +7,20 @@ endif::[]
 [id="{p}-{page_id}"]
 = Uninstall ECK
 
-Before uninstalling the operator, remove all Elastic resources in all namespaces:
+To uninstall the operator:
 
+. Remove all Elastic resources in all namespaces:
++
 [source,shell]
 ----
 kubectl get namespaces --no-headers -o custom-columns=:metadata.name \
   | xargs -n1 kubectl delete elastic --all -n
 ----
++
+This deletes all underlying Elasticsearch, Kibana, and APM Server resources like pods, secrets, services, and so on.
```

Comment:
I think we probably want capitalized k8s resources: `Pods`, `Secrets`, etc. but that's something we can tackle separately.

Reply:
Thanks @sebgl, added initial caps.